package.json

```diff
@@ -12,6 +12,7 @@
     "@angular/platform-browser-dynamic": "9.1.9",
     "@angular/router": "9.1.9",
     "@ngx-translate/core": "12.1.2",
+    "@ngx-translate/http-loader": "^4.0.0",
```

Comment:
remove the ^

Reply:
Already done: https://github.com/3angleTech/webFrame/pull/133/commits/f9265dbb5023645c277bdf8af02f7e9f09da063b